HandEyeCalibration/UtilizeEyeInHandCalibration/UtilizeEyeInHandCalibration.cpp

```diff
@@ -0,0 +1,95 @@
+/*
+Utilize the result of eye-in-hand calibration to transform (picking) point
+coordinates from the camera frame to the robot base frame.
+*/
+
+#include <Eigen/Core>
+#include <Eigen/Dense>
+
+#include <opencv2/core/core.hpp>
+#include <cmath>
+#include <iostream>
+#include <filesystem>
+
+Eigen::MatrixXd cv2eigen(const cv::Mat &cv_mat)
+{
+    if(cv_mat.dims > 2)
+    {
+        throw "Invalid matrix dimensions. Expected 2D.";
+    }
+
+    Eigen::MatrixXd eigen_mat(cv_mat.rows, cv_mat.cols);
```

Comment:
Replace all variables to camelCase instead of snake_case. We use camel for C++ and snake for Python.